tests/rundoctest.py

```diff
@@ -8,12 +8,18 @@
 path = os.path.dirname(__file__)
 if path:
     os.chdir(path)
+os.chdir('..')
```

Comment:
If I run `python test/rundoctest.py` in a freshly cloned repo without installing I get

```
Traceback (most recent call last):
  File "tests/rundoctest.py", line 5, in <module>
    import cypari2
ModuleNotFoundError: No module named 'cypari2'
```
Again, that's because `.` is not in `sys.path` if you run `python script.py`.

Isn't that enough for an error message? Why would you want a tailored message?